### ISSUE TITLE
Avorion server details (./avserver dt) wrong or not applied to server

### ISSUE THREAD

**Issue by kdulep** [GameServerManagers/LinuxGSM#2885]
Avorion server config wrong 
`./avserver dt`
command wrong or wrong code around it:

[https://termbin.com/s8zr](url)

be sure to delete placeholder server config dir /home/avserver/serverfiles/galaxy/avgalaxy/
```
Avorion Server Details
=============================================================================================================================================================================================================================================================================================================================
Server name:      UNAVAILABLE
Server IP:        192.168.1.105:27000
Internet IP:      ****masked***:27000
Server password:  UNAVAILABLE
RCON enabled:     UNAVAILABLE
RCON password:    UNAVAILABLE
Maxplayers:       UNAVAILABLE
Status:           ONLINE
```
```
avserver Script Details
=============================================================================================================================================================================================================================================================================================================================
Script name:            avserver
LinuxGSM version:       v20.2.1
glibc required:         2.15
Discord alert:          off
Slack alert:            off
Email alert:            off
Pushbullet alert:       off
IFTTT alert:            off
Mailgun (email) alert:  off
Pushover alert:         off
Telegram alert:         off
Update on start:        off
User:                   avserver
Location:               /home/avserver
Config file:            /home/avserver/serverfiles/galaxy/avgalaxy/server.ini (FILE MISSING)

```
## Basic info

* **Distro:** CentOS8
* **Game:** Avorion
* **Command:** details, start
* **LinuxGSM version:** [v20.2.1]

## Further Information
details give wrong info, possibly not counting config file
```
avgalaxy="0ZEN"
maxplayers="15"
rconPassword="***masked***"
password="***masked***"


```
## To Reproduce

Steps to reproduce the behaviour:
1. add avgalaxy option to config
2. execute dt command
server starts with 0zen galaxy as expected

## Expected behaviour
Config file:            /home/avserver/serverfiles/galaxy/0ZEN/server.ini 
server password applied to server but not availiable in DT command
rcon password not applied neither showed in DT command
maxplayers not applied to server

server starts with and uses 0zen galaxy as expected in every case just not all config applied and wrong display in ./avserver dt

also 
./avserver  sp
[ WARN ] Stopping avserver: Configuration file missing!
/home/avserver/serverfiles/galaxy/avgalaxy/server.ini

**Comment by dgibbs64** [comment]
I have tested this myself and it is working. Maybe the server didn't download correctly. Try running validate

**Comment by kdulep** [comment]
did u deleted default directory and make server start new galaxy?
after validate same thing

**Comment by dgibbs64** [comment]
I had a fresh install and had no issues at all.

**Comment by kdulep** [comment]
do u have custom named galaxy and not /avgalaxy/

**Comment by dgibbs64** [comment]
no I had everything default

**Comment by dgibbs64** [comment]
I see a possible issue here. give me a few mins

**Comment by dgibbs64** [comment]
I see the issue.
Please add this line under `avgalaxy=`
```
avgalaxypath="${avdatapath}/${avgalaxy}"
```

and let me know if that works

**Comment by kdulep** [comment]
added in avserver.cfg still same

**Comment by dgibbs64** [comment]
try this instead
```
avgalaxy="test"
avgalaxypath="${avdatapath}/${avgalaxy}"
servercfgdir="${avgalaxypath}"
servercfgfullpath="${servercfgdir}/${servercfg}"
```

**Comment by kdulep** [comment]
hmm its uses passwords now but not maxplayers, config dir okish

**Comment by dgibbs64** [comment]
the config default is 10 plays you will need to edit maxplayers.

**Comment by dgibbs64** [comment]
also is there a specific reason why you need to have a specific galaxy name other than the default?

**Comment by kdulep** [comment]
maxplayers="15"

**Comment by kdulep** [comment]
> also is there a specific reason why you need to have a specific galaxy name other than the default?

i just saw galaxy name option i guessed i need fill it , but its not related to server name
is it possible to set server name in games list?

**Comment by dgibbs64** [comment]
Well I ask becuase I was thinking of simply making the galaxy name the name of the linuxgsm script.

e.g `avserver` or `avserver-2` 

and remove the ability to name your own galaxy

**Comment by kdulep** [comment]
yes, but maxplayers and name of server in srver browser?

**Comment by dgibbs64** [comment]
are you setting maxplayers in the server.ini?
```
[Administration]
maxPlayers=10
name=Avorion Server
description=An Avorion Server
password=
pausable=false
```

**Comment by kdulep** [comment]
> are you setting maxplayers in the server.ini?
> 
> ```
> [Administration]
> maxPlayers=10
> name=Avorion Server
> description=An Avorion Server
> password=
> pausable=false
> ```

yes setting also name but its possible to have it in a avserver.cfg

**Comment by dgibbs64** [comment]
no, it's not possible to have it in `avserver.cfg`, you must set it in server.ini. If the variable is not in _default.cfg then it isn't designed to be in `common.cfg` or `avserver.cfg`. The only exception to this is if you set this in start parameters which is not recommended for this server.

**Comment by h3o66** [comment]
There was already a fix released in the version 20.3.0
